### ISSUE TITLE
Misc. Time Warnings, Player FPS, Player Latency, Player Packet Loss and Player Memory No Longer Populate Data after March Rust Update

### ISSUE THREAD

**Issue by jchambo** [Pinkstink-Rust/Rust-Server-Metrics#25]
Hey there - I just set this up and the data looks great, however, the following Rows in Grafana are missing data:

- Misc. Time Warnings
- Player FPS
- Player Latency
- Player Packet Loss
- Player Memory

I spoke with someone who has been using this for quite a while and they noted that ever since the March Facepunch / Rust update, these Rows no longer populate data. Can this be looked into by chance?

![image](https://github.com/Pinkstink-Rust/Rust-Server-Metrics/assets/26009241/b963273f-953e-417b-97d3-10df99183b5f)
![image](https://github.com/Pinkstink-Rust/Rust-Server-Metrics/assets/26009241/9481f8e2-e401-4dd0-8561-d6e35ce378e4)

**Comment by GhilleAU** [comment]
Hey, this was fixed. Make sure you're running the latest version.

**Comment by jchambo** [comment]
@GhilleAU - Hmm, interesting. I used the **RustServerMetrics.dll** from the front page at https://github.com/Pinkstink-Rust/Rust-Server-Metrics ..... Is it possible that link wasn't pointing to the latest release?

**Comment by GhilleAU** [comment]
@jchambo It's working fine on my servers, I would double check to make sure you have the right version.

**Comment by jchambo** [comment]
@GhilleAU - That's odd - I just re-downloaded the **RustServerMetrics.dll** from https://github.com/Pinkstink-Rust/Rust-Server-Metrics/releases/tag/release-v1.5.2 - stopped one of my servers, uploaded it, started it, joined the server but it's still not showing any data for those Rows within Grafana...

Pardon my ignorance, but is there a way I can verify I have the latest release of "everything else" by chance?

My InfluxDB version is **X-Influxdb-Version: 1.8.10** and Grafana is **Grafana v11.0.0 (277ef258d4)**

**Comment by jchambo** [comment]
I figured it out.

I had to download the latest Grafana dashboard file and overwrite the one that existed. That's strange.... I did this install fresh literally last night from all the files from here: https://github.com/Pinkstink-Rust/Rust-Server-Metrics/tree/release-v1.5.2

Looks like it pulled an old Dashboard file somehow? No idea.... but it's working now after I downloaded the Source Files and pulled the Dashboard from **Rust-Server-Metrics-release-v1.5.2.tar.gz\Rust-Server-Metrics-release-v1.5.2\res**
